formbody.d.ts

```diff
@@ -0,0 +1,17 @@
+import * as http from 'http'
+import * as fastify from 'fastify'
+
+declare namespace formBodyPlugin {
+  interface FormBodyPluginOptions {
+    bodyLimit?: number
+  }
+}
+
+declare let formBodyPlugin: fastify.Plugin<
+  http.Server,
```

Comment:
Hi.

I have encountered a type error: "Type 'import("http").Server' is not assignable to type 'import("https").Server'."
Is there a way to use this plugin with https?

Reply:
Interesting, I thought the https.Server extended from http.Server.

Try adding the https parameters as unions to this definition and see if that works:

```ts
declare let formBodyPlugin: fastify.Plugin<
  http.Server | https.Server,
  ...
>
```

IF that works, would you like to open a PR with the updates?